dao/DeliveryServiceRepository.go

```diff
@@ -0,0 +1,37 @@
+package dao
+
+import (
+	"database/sql"
+	"fmt"
+	"log"
+)
+
+type DeliveryServicePostgres struct {
+	db *sql.DB
+}
+
+func NewDeliveryServicePostgres(db *sql.DB) *DeliveryServicePostgres {
+	return &DeliveryServicePostgres{db: db}
+}
+
+type DeliveryService struct {
+	Id          int    `json:"id"`
+	Name        string `json:"name"`
+	Email       string `json:"email"`
+	Photo       string `json:"photo"`
+	WorkingNow  bool   `json:"working_now"`
```

Comment:
what does working now mean? if it is related to time of day, then it should be computed depending on current time, not stored in db. If it is something like if service is accepting orders in general, then it makes sense to call it `status: 'active'/'inactive'/'deleted'`